common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.h

```diff
@@ -68,6 +68,7 @@ class ap_wlan_hal_dwpal : public base_wlan_hal_dwpal, public ap_wlan_hal {
     virtual bool restricted_channels_get(char *channel_list) override;
     virtual bool read_acs_report() override;
     virtual bool read_supported_channels() override;
+    virtual bool update_preference_channels_from_supported_channels() override;
```

Comment:
Please do not add a new API to BWL - this should be abstracted.
Instead, read_preferred_channels should return the supported channels if ACS is disabled, internally in the BWL.

Reply:
removed the commit that adds `update_preference_channels_from_supported_channels`
and in its stead changed the name of the `read_supported_channels()` to `read_preferred_channels()`

If the ACS is disabled the `read_preferred_channels()` will be used to fill the preferred_channels